CHANGELOG.md

```diff
@@ -1,6 +1,6 @@
 # Changelog
 
-## TBD
+## 1.1.0 (2021-12-06)
 
```

Comment:
Worth listing the UTF-8 bug fix: bugsnag/bugsnag-android#1542

Reply:
I've added an entry for this and also for the update to bugsnag-android v4.22.3